packages/hardhat-test-helpers/src/helpers/getStorageAt.ts

```diff
@@ -0,0 +1,24 @@
+import type { NumberLike } from "../types";
+import {
+  getHardhatProvider,
+  assertValidAddress,
+  toRpcQuantity,
+} from "../utils";
+
+export async function getStorageAt(
+  hexAddress: string,
```

Comment:
Here, and in the other methods that receive an address, we should use `address` instead of `hexAddress` as the parameter name. The reason is that it will be a bit clearer/nicer when the user hovers over the function and gets information about the function.